src/components/CPU.js

```diff
@@ -6,6 +6,8 @@ import GotoBox from './CPU/GotoBox';
 import LeftPanel from './CPU/LeftPanel';
 import listeners from '../utils/listeners.js';
 import './CPU.css';
+import {Tab, TabList, TabPanel, Tabs} from "react-tabs";
```

Comment:
nit: I should add to eslint, but prefer `import { a, b, c } from 'z';`, i.e. single quotes and spaces on each side of the braces (only for curlies, not for arrays.)  Not that it's that important...

-[Unknown]

Reply:
No problem, I like when it's consistent too but I'm used to the formatter doing that job. I guess we need those rules:
```
'object-curly-spacing': ['error', 'always'],
'quotes': ['error', 'single'],
'jsx-quotes': ['error', 'prefer-double'],
```